### ISSUE TITLE
Pretrained Model for Inpainting

### ISSUE THREAD

**Issue by jays0606** [eladrich/pixel2style2pixel#1]
Hi, 

  I was wondering if there is a checkpoint for the inpainting model. 

Thank you very much !

**Comment by yuval-alaluf** [comment]
Hi @jays0606! 

Thank you for the interest! 
However, as explained in the paper the inpainting model we illustrated was a "proof of concept" for additional applications and only supports simple triangular masks. Therefore, we don't think releasing a pretrained model would be helpful at the moment.

We do however encourage you to try and train a full inpainting model using pSp and would be happy to share it.

**Comment by jays0606** [comment]
Thank you very much! I'd try it myself and see how it goes.
